### ISSUE TITLE
High Contrast Mode on Windows / IE

### ISSUE THREAD

**Issue by sfrisk** [jquery/css-chassis#36]
Based on this issue in [mobile](https://github.com/jquery/jquery-mobile/issues/6123).  It will be important to keep accessibility in mind when working on Chassis.  From the original issue:

> Set Microsoft Surface to High Contrast Mode by switching the theme in Control Panel\Appearance and Personalization\Personalization . Open http://jquerymobile.com/demos/1.3.0/ in Internet Explorer (in the modern UI). Notice that no icons are shown. There seem to be various recommendations around fixing this:

> http://www.yaccessibilityblog.com/library/high-contrast-friendly-icons.html
> http://msdn.microsoft.com/en-us/library/ie/hh771830(v=vs.85).aspx
> http://www.w3.org/TR/wai-aria-practices/#accessiblewidget (12)

**Comment by trailofdad** [comment]
It would be cool to have documentation on standard practices for accessibility. For example: coding with text-to-speech in mind, and making sure to include alt="" on all images.

**Comment by arschmitz** [comment]
I don't think we want to be a resource on accessibility. There are many good ones already and we should defer to those. This is not our focus when it comes to accessibility. We will make everything we make 100% accessible. And all our demos will show best practices on accessibility. I think thats all we should do for this.

**Comment by trailofdad** [comment]
@arschmitz that makes sense. Thanks for clarifying that.
